cmd/ocsp-updater/main.go

```diff
@@ -421,9 +458,21 @@ func (l *looper) loop() error {
 		if tickEnd.After(expectedTickEnd) {
 			l.stats.Inc(fmt.Sprintf("OCSP.%s.LongTicks", l.name), 1, 1.0)
 		}
-		// Sleep for the remaining tick period (if this is a negative number sleep
-		// will not do anything and carry on)
-		l.clk.Sleep(expectedTickEnd.Sub(tickEnd))
+
+		// After we have all the stats stuff out of the way let's check if the call
```

Comment:
Note this comment is a bit misleading: each call to `tickFunc()` comprises multiple RPC calls. Maybe "let's check if one of the RPC calls failed because..."